arrays.ts

```diff
@@ -62,6 +62,16 @@ export const VERBS: Verb[] = [
   'disassembled',
   'bapped',
   "Blåhaj'd",
+  { word: 'fucked', cw: 'lewd, aggressive' },
+  'added',
+  'subtracted',
+  'modded',
+  'boymoded',
+  'girlmoded',
+  'gender swapped',
+  'rooted',
+  'rotated',
```

Comment:
It seems I accidentally added `rotated` separately and caused the duplicate detection to get upset, sorry about that 😔

```suggestion
```

Reply:
No worries, fixed :3